src/cms/config/index.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs')
+const path = require('path')
+const YAML = require('yaml')
+
+const config = require('../../../gatsby-config.js')
+
+const { defaultLocale, supportedLocales } = config.siteMetadata
+const nonDefaultLocales = supportedLocales.filter(l=>l !== defaultLocale)
+
+function generateConfig() {
+  const configPath = path.join(__dirname, './base-config.yml')
+  const contents = fs.readFileSync(configPath, 'utf8')
+  const base = YAML.parse(contents)
+
+  let newCollections = []
+
+  // if only JS had mapcat
+  base.collections.forEach((collection) => {
```

Comment:
This is dope! 😎 Very cool to programmatically configure the collections for other locales rather than having to manually write it each one in the static/admin/config.yml.

Reply:
:muscle: Tried a couple other approaches (eg a "lang" field) and they were rough, glad this worked out